gsoc/models.py

```diff
@@ -20,24 +20,100 @@ def __str__(self):
         return self.command
 
 
-class SubOrg(models.Model):
+class SubOrg(models.Model):# SubOrg name
     suborg_name = models.CharField(name='suborg_name', max_length=80)
 
     def __str__(self):
         return self.suborg_name
 
 
-class GsocYear(models.Model):
+class GsocYear(models.Model):# Gsoc Year
     gsoc_year = models.IntegerField(name='gsoc_year')
 
     def __str__(self):
         return str(self.gsoc_year)
 
 
+class Reason(models.Model):# Why does your org want to participate
+    reason = models.CharField(name='reason', max_length=250)
+
+    def __str__(self):
+        return self.reason
+
+
+
+class NumMentors(models.Model):# Number of mentors in suborg
+    num = models.IntegerField(name='num')
+
+    def __str__(self):
+        return str(self.num)
+
+
+class SubOrgAdminEmail(models.Model):# SubOrg Admin Email
+    suborg_admin_email = models.EmailField(name='suborg_admin_email')
+
+    def __str__(self):
+        return str(self.suborg_admin_email)
+
+
+class MentorEmail(models.Model):# Mentor Email
```

Comment:
There will be more then one of these fyi, you could do it this way but you need some way to reference it back then.